pep_sphinx_extensions/pep_zero_generator/parser.py

```diff
@@ -205,3 +213,39 @@ def _parse_author(data: str) -> list[tuple[str, str]]:
         if author_list:
             break
     return author_list
+
+
+def _abbreviate_type_status(type_: str, status: str) -> str:
+    """Add tooltip for the PEP type and status"""
```

Comment:
```suggestion
    """Add tooltip for the PEP type and status."""
```

Trivial comment (if you apply the suggestion,make sure the `Co-authored-by` is either rebased out or elided in the final squashed commit description): Docstring summary lines should end with a period (as a sentence), per PEP 257 and convention here.